site/docs/samples/notebook-read.md

```diff
@@ -36,13 +36,17 @@ Make a note of the service endpoint, bucket name, and access credentials. You wi
     === "Kubernetes"
             helm repo add localstack-charts https://localstack.github.io/helm-charts
             helm install localstack localstack-charts/localstack \
+                 --version {{LOCALSTACK_CHART_VERSION}} \
+                 --set image.tag="{{LOCALSTACK_VERSION}}" \
                  --set startServices="s3" \
                  --set service.type=ClusterIP \
-                 --set livenessProbe.initialDelaySeconds=25
+                 --set livenessProbe.initialDelaySeconds=25 
```

Comment:
pls remove the `space` at the end of the line

Reply:
Done